### PR TITLE
[WIP] Add some what's new

### PR DESCRIPTION
**Pull request by Carreau**
Went though PR and collected new feature.

### CODE REVIEW

File: docs/source/whatsnew/pr/newfeat.rst [hunk 1]
```diff
@@ -0,0 +1,14 @@
+YouTube video will now show thumbnail when exported to a media that do not support video. (PR #9086)
```
Comment (takluyver) [inline docs/source/whatsnew/pr/newfeat.rst]: Do you want to use the `:ghpull:` role here too?

File: docs/source/whatsnew/pr/newfeat.rst [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+Add warning when running `ipython <subcommand>` when subcommand is deprecated. `jupyter` should now be used.
+
+Code in `%pinfo` (also known as `??`) are now highlighter (:ghpull:`#8947`)
```
Comment (takluyver) [inline docs/source/whatsnew/pr/newfeat.rst]: Looking at the source of the github extension, I think these need to be without the `#`. I might be wrong,  though.

File: docs/source/whatsnew/pr/newfeat.rst (hunk 1 above)
Comment (Carreau) [inline docs/source/whatsnew/pr/newfeat.rst]: done

File: docs/source/whatsnew/pr/newfeat.rst (hunk 2 above)
Comment (Carreau) [inline docs/source/whatsnew/pr/newfeat.rst]: done

**Pull request merged**